### ISSUE TITLE
Does not affect clock (much)

### ISSUE THREAD

**Issue by ardee00** [marticliment/ElevenClock#1395]
### Discussed in https://github.com/marticliment/ElevenClock/discussions/1394

<div type='discussions-op-text'>

<sup>Originally posted by **ardee00** December 26, 2024</sup>
I can run ElevenClock fine on my Windows 11 Pro 24H2 system. I made numerous changes to settings, such as using a larger font for the clock. But the changes never take effect (except a small artifact to the right of the clock) -- even after restarting Explorer or restarting the machine. Do you know of an issue coexisting with Classic Shell?</div>

**Comment by marticliment** [comment]
Do you have "Protected folder access" enabled on Microsoft Defender?

**Comment by ardee00** [comment]
No, “Controlled folder access” is disabled in the “Ransomware protection” section of my Windows 11 Pro 24H2 installation.

**Comment by marticliment** [comment]
Can you see anything unexpected on ElevenClock logs (ElevenClock Settings -> Debbugging info -> Log)

**Comment by ardee00** [comment]
I don't think so. Here's the full log:

🟡 Using bundled lang file (forced=False)
🟡 Using bundled lang file (forced=True)
It took 0.008014202117919922 to load all language files
---------------------------------------------------------------------------------------------------

   ElevenClock's 4.4.1.1 (v4.411) log: Select all the text and hit Ctrl+C to copy it
   All modules loaded successfully and sys.stdout patched correctly, starting main script
   Translator function set language to "en"

---------------------------------------------------------------------------------------------------

 Log legend:
 🔵: Verbose
 🟢: Information
 🟡: Warning
 🟠: Handled unexpected exception
 🔴: Unhandled unexpected exception
 🟣: Handled expected exception

🔵 Initializing clock 0...
🔵 Using taskbar background color
🟢 Clock on the bottom
🟢 Clock on the right
🔵 Clock geometry: PySide6.QtCore.QRect(3015, 1681, 57, 46)
🔵 Full screen rect:  (0, 0, 3072, 1728)
🔵 Using automatic font color
🔵 Font families   : []
🔵 Custom font     : <PySide6.QtGui.QFont(8514oem,15,-1,5,400,0,0,0,0,0,0,0,0,0,0,1) at 0x00000149106B3A40>
🔵 Font size: 10.0
🟡 Desktop button enabled
🔵 Loaded date time format: %H:%M:%S  %p\n%#m/%#d/%Y  (clock 0
🔵 Show/hide loop started with parameters: HideonFS:True, NotHideOnTB:False, DisableNotifications:True
🟢 Loaded clocks in 0.8534681797027588
🔵 Restart loop: 1
🟠 Update checking disabled
🔵 Low cpu mode is set to False. DisableNotifications is set to False
🟢 Loaded everything in 1.5253150463104248
🔵 Initializing clock 0...
🔵 Using taskbar background color
🟢 Clock on the bottom
🟢 Clock on the right
🔵 Clock geometry: PySide6.QtCore.QRect(3015, 1681, 57, 46)
🔵 Full screen rect:  (0, 0, 3072, 1728)
🔵 Using automatic font color
🔵 Font families   : []
🔵 Custom font     : <PySide6.QtGui.QFont(8514oem,15,-1,5,400,0,0,0,0,0,0,0,0,0,0,1) at 0x00000149106B3A40>
🔵 Font size: 10.0
🟡 Desktop button enabled
🔵 Loaded date time format: %H:%M:%S  %p\n%#m/%#d/%Y  (clock 0
🔵 Show/hide loop started with parameters: HideonFS:True, NotHideOnTB:False, DisableNotifications:True
🔵 Announcement URL: https://www.marticliment.com/resources/elevenclock.announcement
🔵 Image URL: https://marticliment.com/resources/translate.png
🔵 Restart loop: 1
🔵 Saving log...
🟠 Update checking disabled

**Comment by marticliment** [comment]
Disabling classic shell allows the clock to work again?

**Comment by ardee00** [comment]
Hello Marti,

No, disabling ClassicShell seems to make no difference.

Here’s what my tray clock looks like when ElevenClock is running with a few customization (larger, bolder font; “ PM” changed to “p”):



The rightmost portion of the clock gets overwritten by the bold “p” and “5” and the extreme-right area works to minimize all windows.

Does that give you a clue?

Thank you!

  -Roy.

PS: This is on a fresh Windows 11 Pro 24H2 installation done last month.

**Comment by marticliment** [comment]
Hello Roy, I can't see the attached images. GitHub does not support email-sent images, you will need to attach them directly on GitHub

**Comment by ardee00** [comment]
Sorry about that. I'll try pasting it here in GitHub:
![image](https://github.com/user-attachments/assets/d7dd1da1-4dcf-40b6-9f36-57a06bf4e5ec)

**Comment by marticliment** [comment]
Have you changes the "Change the width of the clock" setting on ElevenClocl Settings?

**Comment by ardee00** [comment]
Hi. Changing width, height, or horizontal position has no effect. It still looks the same as that picture I posted. ☹

**Comment by marticliment** [comment]
This is really weird, from the logs every looks normal, I have never seen this behaviour.
I will investigate it and update you on anything I can find

**Comment by ardee00** [comment]
Thank you very much Marti!
